quantum/action_tapping.c

```diff
@@ -184,11 +184,11 @@ bool process_tapping(keyrecord_t *keyp) {
                             (
                                 (
                                     false
-#            if defined(HOLD_ON_OTHER_KEYPRESS) || defined(HOLD_ON_OTHER_KEYPRESS_PER_KEY)
+#            if defined(HOLD_ON_OTHER_KEY_PRESS) || defined(HOLD_ON_OTHER_KEY_PRESS_PER_KEY)
                                     || (
                                         IS_LT(tapping_keycode)
-#                ifdef HOLD_ON_OTHER_KEYPRESS_PER_KEY
-                                        && get_hold_on_other_keypress(tapping_keycode, keyp)
+#                ifdef HOLD_ON_OTHER_KEY_PRESS_PER_KEY
+                                        && get_HOLD_ON_OTHER_KEY_PRESS(tapping_keycode, keyp)
```

Comment:
```suggestion
                                        && get_hold_on_other_key_press(tapping_keycode, keyp)
```
Looks like the testing matrix needs to be expanded to have `HOLD_ON_OTHER_KEY_PRESS_PER_KEY` too.